test/acceptance/features/steps/namespace.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+from command import Command
+
+
+class Namespace():
+    def __init__(self, name):
+        self.name = name
+        self.cmd = Command()
+
+    def create(self):
+        create_namespace_output, exit_code = self.cmd.run("oc new-project {}".format(self.name))
```

Comment:
In the makefile, you've `set-namespace` target which runs `oc new-project` and here too, are they creating different projects?

Reply:
@pratikjagrut `set-test-namespace` is used to set (switch to) the namespace for `oc` tool. If you look closely, the target actually have `oc project <..>` and not `oc new-project <...>` so it does not create a new namespace.

Reply:
Okay, I get it.